src/Record/Record.Model/FileRecordBuilder.cs

```diff
@@ -27,15 +29,25 @@ private record Storage
     }
 
     #region With-Methods
-    public FileRecordBuilder WithId(Uri id) =>
+    public FileRecordBuilder WithId(IriReference id) =>
```

Comment:
Do I remember correctly that this method would also work if the user had an Uri? I think I remember an implicit converter from Uri to IriReference.

Reply:
Yes it does. I will add a test that checks it.